styles.js

```diff
@@ -10,5 +11,54 @@ export default createGlobalStyle`
   body {
     margin: 0;
     font-family: system-ui;
+    color: #2F3C7E;
+    background-color: #FBEAEB;
+    padding: 30px;
   }
+
+  h2 {
+  font-size: 1.4em;
+  background-color: lightblue;
+  border: 2px groove black;
+  border-radius: 4px;
+  text-align: center;
+  
+  }
+
+  h1 {
+    text-align: center;
+    font-size: 3.5em;
+  }
+  Image {
+    justify-content: center;
+  }
```

Comment:
you can extend the Image component with styled components and should do so.

```suggestion
```

see [this as reference](https://github.com/neuefische/web-bo-22-2/blob/main/sessions/react-styled-components/react-styled-components.md#styling-a-custom-component)